lab_1/main.py

```diff
@@ -7,3 +7,5 @@ def say_hello_world() -> None:
     """
     Say hello world.
     """
+print("hello")
```

Comment:
Looks like you have strange repository state, We removed this folder from the main repo. Please, ask me on the next seminar to help you.